docs/advanced_topics/images/custom_image_model.md

```diff
@@ -40,9 +40,12 @@ class CustomRendition(AbstractRendition):
     image = models.ForeignKey(CustomImage, on_delete=models.CASCADE, related_name='renditions')
 
     class Meta:
-        unique_together = (
-            ('image', 'filter_spec', 'focal_point_key'),
-        )
+       constraints = [
+            models.UniqueConstraint(
+                fields=['image', 'filter_spec', 'focal_point_key'],
+                name='rendition_unique',
```

Comment:
I think we tend to put the `unique` in front of the constraint name:

https://github.com/wagtail/wagtail/blob/09c7f66e03621da042e9faf5c9be2a74ff2f20cd/wagtail/models/__init__.py#L3257-L3262

https://github.com/wagtail/wagtail/blob/09c7f66e03621da042e9faf5c9be2a74ff2f20cd/wagtail/models/__init__.py#L4496-L4502

```suggestion
                name='unique_rendition',
```